Mathlib/Order/RelIso/Basic.lean

```diff
@@ -133,10 +138,12 @@ protected def comp (g : s →r t) (f : r →r s) : r →r t :=
   ⟨fun x => g (f x), fun h => g.2 (f.2 h)⟩
 
 /-- A relation homomorphism is also a relation homomorphism between dual relations. -/
+@[simps]
 protected def swap (f : r →r s) : swap r →r swap s :=
   ⟨f, f.map_rel⟩
 
 /-- A function is a relation homomorphism from the preimage relation of `s` to `s`. -/
+@[simps]
 def preimage (f : α → β) (s : β → β → Prop) : f ⁻¹'o s →r s :=
   ⟨f, id⟩
```

Comment:
This doesn't generate the `coe_` lemmas right? Shouldn't we have them?

Reply:
I don't think they're that important; someone can add them if/when they need them.